CHANGELOG.md

```diff
@@ -1,3 +1,20 @@
+# Release 0.25.13
+
+## What's New
+- Edge
+  - Bug fixes
+- Fabric
+  - N/A
+- Ziti CLI
+  - N/A
+- SDK Golang
+  - N/A
+
+## Edge
+### Bug Fixes
+
+* https://github.com/openziti/edge/issues/1055
```

Comment:
can you pleease describe the bug on the notes. save me (or other readers) the click?

Reply:
Added a note